Mage.Sets/src/mage/cards/c/Conspiracy.java

```diff
@@ -137,7 +138,8 @@ private void setCreatureSubtype(MageObject object, SubType subtype, Game game) {
             if (object == null) {
                 return;
             }
-            SubTypes subTypes=game.getState().getCreateMageObjectAttribute(object, game).getSubtype();
+            SubTypes subTypes = game.getState().getCreateMageObjectAttribute(object, game).getSubtype();
+            subTypes.setIsAllCreatureTypes(false);
```

Comment:
Same `removeAllCreatureTypes`

Reply:
there's no method for that one

Reply:
But `removeAllCreatureTypes` make same code: takes subtypes from gameState's attributes and change it:

![shot_210125_062405](https://user-images.githubusercontent.com/8344157/105653733-fb2ce300-5ed5-11eb-9a28-965669db25ad.png)

Reply:
Hmm, it's different:

![shot_210125_062750](https://user-images.githubusercontent.com/8344157/105653973-78585800-5ed6-11eb-870a-5f5843442e96.png)

Reply:
right, but `getSubtype` doesn't call `getCreateMageObjectAttribute` so it affects the card directly rather than the `MageObjectAttribute`

Reply:
yeah, exactly. I'll try changing it and see if the tests all pass

Reply:
`removeAllCreatureTypes` will be "bugged" if no `MageObjectAttribute` exist (will remove types and all status from the object forever).

Reply:
yeah I think it needs to be `getCreateMageObjectAttribute` instead of `getMageObjectAttribute`, I'll let you know in like 10 minutes when the tests finish running

Reply:
I think `removeAllCreatureTypes(boolean, game)` must be fixed to use getCreateMageObjectAttribute (e.g. works with attributes all the time). Is that true?

Reply:
I think `getSubtype` needs to use it

Reply:
General logic for game methods: 
* if you call it with `game` param when it's a dynamic data (e.g. can be affected by game effects). 
* if you call without `game` param when it's a persistent data from that object only.

CardAttributes creation code require original object's subTypes, so there is must be method without game param like `getSubtypes()`.

Reply:
More logic:
* use getCreateMageObjectAttribute for write operations (setSubtype, setAllCreatureType)
* use getMageObjectAttribute for read read operations like `getSubtypes(game)`

P.S. Also you must check all `getsubtype(game).remove` and `getsubtype(game).add` usage and fix it.

Reply:
Also don't forget what all `mageObjectAttribute` are reset on each game cycle, so you can change attribute related data from `ContinuousEffect` only (e.g. one shot effects can't change color or subtypes).